src/components/common/CardSignInRequired.vue

```diff
@@ -1,25 +1,107 @@
 <template>
-  <TmDataMsg icon="account_box">
-    <div slot="title">
-      Sign In Required
-    </div>
-    <div slot="subtitle">
-      This page shows data for a specific address. To access the page, please
-      <a @click="$router.push(`/welcome`)">sign&nbsp;in</a> or
-      <a @click="$router.push(`/welcome`)">sign&nbsp;up</a>.
+  <div class="card-sign-in">
+    <h2>Welcome to Lunie</h2>
+    <h3>How would you like to get started?</h3>
+
+    <div class="session-list">
```

Comment:
this now creates a lot of dupe code

Reply:
i can delete the other components where this lived originally `TmSessionExisting`

...and a refactor is in order, but I'm trying to deliver the value of a friendlier home screen for now.

Reply:
if you want to defer this, please create a refactoring issue and let's do it soon.

Reply:
> this now creates a lot of dupe code

no it doesn't, I moved this from `TmSessionExisting` and will delete it!